app.rb

```diff
@@ -0,0 +1,82 @@
+require 'sinatra'
+require_relative 'lib/hangman_rules'
+require_relative 'lib/word_generator'
+require_relative 'lib/words'
+require_relative 'lib/word_formatter'
+
+class App < Sinatra::Base
+
+  enable :sessions
+
+  helpers do
+
+    def rules
+      HangmanRules.new(secret_word, guesses)
+    end
+
+    def letters
+      if session[:letters].nil?
+	session[:letters] = ("a".."z").to_a
+      end
+      session[:letters]
+    end
+
+    def secret_word
+      if session[:secret_word].nil?
+	session[:secret_word] = WordGenerator.new(Words.new).random_word
```

Comment:
👍 I think that's good usage of another class here. Basically all the logic for generating the word is in that other class and all you're doing here is working with the session.

Reply:
Agree